packages/react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -91,7 +91,9 @@ export const Tooltip: React.FC<ITooltipProps> = (props) => {
     } else {
       visible && onClose?.();
```

Comment:
shouldn't it fire regardless of the fact that tooltip is managed or not?

Reply:
I've modified this component, so callback will be fired whenever `isVisible` value is changing